### PR TITLE
Added portuguese translation

### PR DESCRIPTION
**Pull request by swyknox**
I realized that Eyedropper didn't have a translation for my language, Portuguese, so I did it. I ran some tests and everything seems to be going well.

I've never opened a PR, so I don't know if I'm doing it the right way, I hope so.

About maintenance of the translation, I can't say for sure if i'm be available at other times, however, if I notice any updates with the addition of new strings, I'll take a look and see what I can do!

preview

### CODE REVIEW

File: po/pt.po [hunk 1]
```diff
@@ -0,0 +1,739 @@
...
+msgctxt ""
+"CIE illuminant, more info at https: en.wikipedia.org wiki Standard_illuminant"
+msgid "F11 (Ultralume 40, Philips TL84)"
+msgstr ""
+
+#: data/resources/ui/preferences.ui:112
+msgctxt ""
+"CIE illuminant, more info at https: en.wikipedia.org wiki Standard_illuminant"
+msgid "F12 (Ultralume 30, Philips TL83)"
+msgstr ""
```
Comment (FineFindus) [inline po/pt.po:330]: While the names themselves should not be translated, their description, such as `Daylight` should be.

File: po/pt.po [hunk 2]
```diff
@@ -0,0 +1,739 @@
...
+msgstr "Copiar código LMS"
+
+#: data/resources/ui/window.ui:213
+msgid "Copy Hunter Lab"
+msgstr "Copiar código Hunter Lab"
+
+#. Translators: This should not be translate, Please enter your credits here instead (format: "Name https://example.com" or "Name <email@example.com>", no quotes)
+#: src/widgets/about_window.rs:51
+msgid "translator-credits"
+msgstr "swyknox"
```
Comment (FineFindus) [inline po/pt.po:552]: While this is technically not wrong, are you sure that you do not want to link an email address or website (or your GitHub account)?

**Review by FineFindus — CHANGES_REQUESTED** [review]
Thanks. I found two minor issues, that should be addressed before merging.

It's completely fine if you do not have the time to update the strings, I usually send an E-Mail before creating a release, so you do not have to permanently watch for changes.

File: po/pt.po (hunk 2 above)
Comment (swyknox) [reply po/pt.po]: Well thought out, I added my email!

**Comment by FineFindus** [comment]
Thanks!

**Review by FineFindus — APPROVED** [review]

**Pull request merged**